peregrine/resources/submission/graphql/util.py

```diff
@@ -92,7 +92,8 @@ def authorization_filter(q):
     """
 
     try:
-        fg.read_access_projects = fg.get('read_access_projects') or fg.user.get_project_ids('read')
+        if not hasattr(fg, 'read_access_projects'):
```

Comment:
Why does this need to be changed?

Reply:
https://github.com/uc-cdis/peregrine/blob/master/peregrine/resources/submission/__init__.py#L36 should be used for this instead of calling the user function because this includes the open access projects. Probably we don't need this code at all? I don't think any of the code paths take you into this function without calling set_read_access_projects first but I left it anyways for now